CHANGELOG.md

```diff
@@ -31,6 +31,7 @@
 
 - Update caniuse database ([#1046](https://github.com/opensearch-project/oui/pull/1046))
 - Bump TypeScript to v4.6.4 ([#879](https://github.com/opensearch-project/oui/pull/879))
+- Cleaning up of react-datepicker([#1067](https://github.com/opensearch-project/oui/pull/1067))
```

Comment:
To keep the language consistent with other change log messages, I would suggest saying "Clean up react-datepicker".

Reply:
```suggestion
- Clean up `react-datepicker` package to remove unnecessary directories and files ([#1067](https://github.com/opensearch-project/oui/pull/1067))
```

Reply:
The last suggestion from @joshuarrrr has been incorporated in CHANGELOG.md